### ISSUE TITLE
Continue Use of DialogBlocks Tool?

### ISSUE THREAD

**Issue by tomspilman** [SickheadGames/Torsion#13]
I originally used a tool called DialogBlocks (http://www.dialogblocks.com/) for generating all the window/dialog layouts in Torsion.  You can see the project file for it here:

 https://github.com/SickheadGames/Torsion/blob/master/code/Torsion/Torsion.pjd

And how the dialog files have a warning about them being generated by this tool...

 https://github.com/SickheadGames/Torsion/blob/master/code/Torsion/BreakpointPropertyDlg.cpp#L5

The question is if we should keep using this tool moving forward.  It seems as though the tool is still supported and maintained with a release in 2015:

 http://www.dialogblocks.com/download.htm

It seems as the free demo version was all we needed for development in Torsion, but i seem to remember having a licenses way back then.

The alternative is hand writing dialogs in the blind... and while 10 years ago i wasn't very good at that...  it wouldn't bother me too much as i don't see us needing to do a lot of that.

Thoughts?

**Comment by dottools** [comment]
I've used [wxFormBuilder](https://sourceforge.net/projects/wxformbuilder/files/wxformbuilder-nightly/) for my wxWidgets projects with really good results.

Only real problem I've had with it is that the previewing of forms/dialogs feature doesn't produce the same results you get when you build and run your program, such as resizing issues show-up in preview, while it all works fine the real C++ project program its self for example.

**Comment by tomspilman** [comment]
Yeah...  i'm so far removed from those tools at this point.  Been spoiled by the C# ecosystem and don't really know what the pitfalls of wxFormBuilder or DialogBlocks are anymore.

As long as we don't have a need to build a ton of new dialogs it probably doesn't matter if we stop using the tools.  And for new dialogs we can use the tool to create it then stop using it if we want.
